### ISSUE TITLE
Add data about age restrictions for hormones pages

### ISSUE THREAD

**Issue by fallax** [genderkit/genderkit#114]
We need to indicate the situation for younger people on pages about surgery and hormones. Starting points for researching this information might be:

- the websites of the UK gender services that serve young people (see https://genderkit.org.uk/resources/gender-services/)
- the treatment protocol for young people in the UK (can be found here https://genderarchive.org.uk/tag/treatment-protocols/)

This information needs to apply specifically to the rules on clinics **based in the UK**.

Information about pages needs to be put in a consistent format into the files containing data for each article that can be found at ```_data/articles/*.yaml``` so that it can be read out by the changes to implement issue #101

**Comment by jlphackett** [comment]
From https://www.england.nhs.uk/commissioning/wp-content/uploads/sites/12/2016/08/clinical-com-pol-16046p.pdf we have
> Cross sex hormones may be prescribed to younger people with gender dysphoria from around their 16th birthday subject to individuals meeting the eligibility and readiness criteria, noting that in exercising this discretion, the specialist multi- disciplinary team will be cognisant that international guidelines recommend the prescribing of cross sex hormones to gender variant young people at “around 16 years”.

This is specific to NHS England but the other nations do not seem to have specific protocols for young people, and in the case of Scotland and Wales they refer to Tavi anyway. Not sure what NI do as I have not yet found any protocol documents.

WRT blockers, the GIDS service spec states:

> Clients under the age of 16 should be assessed regarding their ability to give informed consent and whether they have appropriate autonomy to make decisions.
> The decision to start hormone blockers is reached after an in-depth discussion involving the MDT, following which the final responsibility for prescribing the hormone blocker and the physical monitoring of this treatment remains with the paediatric and adolescent endocrinologists, subject to periodic review by the MDT.

They then list criteria for referring to the paediatric endo, those being:
> * the adolescent has been presenting with continuing GD and the intensity and distress has increased with puberty
> * the adolescent presents as relatively stable psychologically as evaluated through clinical
observation and questionnaires;
> * there is support from the family/carers;
> * where there is a need to provide information about physical development in order to allay
some anxieties in the adolescent patient and the family;
> * to exclude a disorder of sex development (intersex) or other endocrine conditions;

Re the paediatric endo, we have:

> Physical intervention in the early stages of puberty is available via the Paediatric Endocrine Liaison Team’s Early Intervention’ Clinic for carefully selected clients who are at least in Tanner Stage 2 of puberty and are up to the age of 15. The Early Intervention Clinic will continue to follow the Service’s 2011 research protocol, which following evaluation, has now become established practice, with the exception that hormone blockers will now be considered for any children under the age of 12 if they are in established puberty.

The 2011 research protocol seems to be a reference to [this](https://gids.nhs.uk/our-early-intervention-study). The protocol itself is included in [this FOIA response](https://tavistockandportman.nhs.uk/documents/1544/FOI_18-19398_Paper_on_adolescents_with_Gender_Identity_Disorder.pdf), which (in turn) states that decisions are based on the criteria used by the Amsterdam Gender Clinic. According to the first source, patients with abnormally low BMI are excluded, and those with abnormally low bone density must give informed consent to the risks.

The PELT's evaluation will involve a physical exam to determine Tanner Stage.

The above applies to those under the age of 15. For those over 15 who are post-puberty, the PELT is not involved and the criteria are:
> * there is a substantial history of gender incongruence, lasting more than one year. While a diagnosis of Gender Dysphoria (DSM-5) may be made after six months of dysphoria, a move to physical treatment requires a further period of consolidation.
> * the young person is judged to have sufficient understanding of what the blocker will do, and how it works, to be able to give assent, or consent, to treatment.
> * if the request for blockers seems driven by a wish for no puberty or no gender, perhaps accompanied by a generalised dissatisfaction with the body, and this motivation has been carefully explored. In such a case, the young person and their parents/carers have an understanding of the limitations of what medical intervention can offer in the longer term.
> * similarly, if the request, and any accompanying distress, seem linked to sexual orientation rather than gender identity, and the motivation has been carefully explored.
> * there is no intense and prolonged psychological illness on the part of the young person (such as a severe eating disorder, psychotic experiences or major depression) such as might interfere with considered decision-making by the young person.
> * there is no ongoing major family disruption.
> * one or both parents/carers support the young person’s request for puberty suppression treatment and work has been done to develop their understanding of its potential advantages and its potential disadvantages.
> * where the parents are separated, it has been established who has legal parental responsibility, and careful thought has been given to involving an estranged parent in the decision-making about the young person's treatment.
> * the young person and family are likely to be able to attend appointments regularly. Any likely barriers to attending have been explored prior to referral for puberty suspension.
> * the young person is engaged in education and some face to face social interaction with peers.
> * the young person is at least at Tanner Stage 2 i.e. ‘in puberty’.

For surgery, we have:
> If individuals fulfil additional criteria, they may have various types of gender affirming surgery from the age of 18 through adult gender identity clinics.
In other words, GIDS don't do surgery referrals.

So as you can plainly see the situation is not at all complicated or involved.

Summary
=======
* Under 16s don't get hormones, but can get blockers.
* Blockers for under 15s requires psychological stability, support of family/carers and continuing dysphoria (and the distress has increased with the onset of puberty), as well as being in Tanner Stage 2 or later. There are also some criteria around body mass index and bone density.
* Blockers for 15+ requires psychological stability, support of one parent and continuing (>= 1 year) dysphoria, as well as being in Tanner Stage 2 or later, having a functioning social life of some kind (lolcovid) and being able to give informed consent.
* Surgery is for 18+ only.

**Comment by jlphackett** [comment]
Brackenburn's _Knowing Our Identity_ (NI) states:
* Patients can also currently access hormonal treatments from age 15 and up, subject to assessment.

**Comment by jlphackett** [comment]
Splitting the surgery off to its own issue #125
